### ISSUE TITLE
Focus indicator is not clearly visible on the Menu button.

### ISSUE THREAD

**Issue by davidstenger** [18F/web-design-standards#1655]
**Issue**:
The focus indicator is not visible when user navigates to the Menu Button. Having the clear visual focus is critical for a sighted keyboard user to track where they are on the page. 
See attachment [ResponsiveViewMenuFocus_ExtendedHeader.png](https://drive.google.com/open?id=0ByQeU_iOqP_iZzhIQk9VWDkzS28)

**Solution**:
Properly apply the focus indicator to the Menu button so that the user may be able to track keyboard focus.

**Comment by toolness** [comment]
@rtwell do you have any suggestions on this one?  

![responsiveviewmenufocus_extendedheader](https://user-images.githubusercontent.com/124687/28325372-a34003d2-6bab-11e7-8c39-14a9d03e61fd.png)

We're providing the same focus styling on the menu button that we apply to other parts of the site; however, the problem here is that the menu button itself is the same shade of blue as the focus ring, so it's actually hard to tell that the menu button is the current focus target.  Basically the only visual differentiation is that the menu button has a fuzzy border now instead of a straight one.

Any suggestions on how we might better indicate that the menu button is focused, without breaking visual consistency?  One solution, for example, might be to apply an underline effect to the "MENU" text, or we could change the focus ring color for just this one button...

**Comment by rtwell** [comment]
my suggestion would be to implement a design that considers the color of the component , which would be new functionality we bake into the WDSs. For example, all primary blue buttons receive a focus style XYZ, whilst secondary red buttons receive a different style ABC.

**Comment by toolness** [comment]
Cool, that sounds good to me! I think that there is a general guideline to try to keep the focus style as consistent as possible, so that folks can just e.g. "look for the pink rectangle" or whatever whenever they press <kbd>Tab</kbd>, but that might not be possible with the Standards.

**Comment by rtwell** [comment]
i'm thinking we could shave off a few of our button choices, and that would
make the focus issue easier to solve

**Comment by donjo** [comment]
This was addressed via #2112 and the new focus state is a grey dotted line that should be a bit easier to see against the blue menu button. There is also a `focus-outline` variable in `_variables.scss` that can be modified to fit different background styles.
